Dockerfile

```diff
@@ -1,11 +1,23 @@
-FROM alpine:latest
+FROM registry.access.redhat.com/ubi8-minimal:latest
 MAINTAINER Portworx Inc. <support@portworx.com>
 
-RUN apk add tar python3 ca-certificates && apk upgrade
+ARG VERSION=master
```

Comment:
Won't this overwrite whatever is passed in from the Makefile?

Reply:
if VERSION is not passed via build-arg use `master`